controllers/imagejob/imagejob_controller.go

```diff
@@ -108,10 +125,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, err
 	}
 
-	// map of node names and runtime
-	nodeMap := processNodes(nodes.Items)
-
-	for nodeName, runtime := range nodeMap {
+	for _, n := range nodes.Items {
+		n := n
```

Comment:
don't need this

Reply:
I think we may need this since we are taking a pointer later on.
gosec linter should flag.

Reply:
Yes that's right